test/test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+require('mocha');
+require('should');
```

Comment:
actually it is being used on the first unit test. still, I'll go ahead and remove it